### ISSUE TITLE
[Request] whitespace character

### ISSUE THREAD

**Issue by ChangJoo-Park** [careteditor/caret#355]

**Comment by astoilkov** [comment]
Hi @ChangJoo-Park,

Can you explain a little more about this request. Thanks in advance.

**Comment by manuelbua** [comment]
@ChangJoo-Park do you mean the option to actually see whitespace characters, such as tabs, spaces and indent guides?

**Comment by pslobo** [comment]
I think so. Personally I don't see a need for that in a text editor. In a code editor yes, but Caret isn't a code editor so I think that would simply be visual clutter and not bring much benefit.

**Comment by pslobo** [comment]
Fair point. If this is implemented I'd request it be toggleable in settings.
